### PR TITLE
modify pvc-autoresizer

### PR DESCRIPTION
**Pull request by buffalo1024**
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/pvc-autoresizer/pvc_informerV2.go [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+}
+
+func (w *pvcWorker) conformPVCAgainstResizePolicy(pvc *corev1.PersistentVolumeClaim, rp *hwameistorv1alpha1.ResizePolicy) (bool, error) {
+	if rp.Spec.PVCSelector != nil {
+		matched, err := matchPVC(rp, pvc)
+		if err != nil {
+			log.Errorf("match pvc err: %v", err)
+			return false, err
+		}
+		if !matched {
```
Comment (sun7927) [inline pkg/pvc-autoresizer/pvc_informerV2.go:33]: why return false? should it continue to be checked more?

**Comment by sun7927** [comment]
In this solution, once a PVC gets updated, it will be checked against the policy and associated with a policy if matched. In case, user wants to disable auto resize for a PVC, how to handle it?

File: pkg/pvc-autoresizer/pvc_informerV2.go (hunk 1 above)
Comment (buffalo1024) [reply pkg/pvc-autoresizer/pvc_informerV2.go:33]: it's && relation between selectors of a resizepolicy, so if one selector doesn't match, the whole resizepolicy doesn't match.

**Comment by buffalo1024** [comment]
> In this solution, once a PVC gets updated, it will be checked against the policy and associated with a policy if matched. In case, user wants to disable auto resize for a PVC, how to handle it?

to disable autoresizing for a pvc,  keep that no resizepolicy select the pvc.

**Pull request merged**